bundler/lib/bundler/source/git.rb

```diff
@@ -80,7 +80,7 @@ def to_s
           ""
         end
 
-        "#{@safe_uri}#{rev}"
+        "#{@safe_uri}#{rev}#{glob_for_display}"
```

Comment:
I'm not 100% sure of the format, this makes it so that both annotations show up independently e.g.

```
Using foo 0.1.1 from git@github.com:org/repo.git (at f334c1b@f334c1b) (glob: gems/foo/*.gemspec)
```
Maybe we should have a single set of parenthesis and separate them by comma instead

```
Using foo 0.1.1 from git@github.com:org/repo.git (at f334c1b@f334c1b, glob: gems/foo/*.gemspec)
```

Reply:
Both options seem fine to maybe, but if I have to choose I think I'd go with the compact version although not a big deal.